### PR TITLE
add 8.1.0 performance test

### PR DESCRIPTION
**Pull request by shaoxiqian**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Yui-Song — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: tidb-cloud/v8.1-performance-benchmarking-with-sysbench.md
Comment (Oreoxmt) [inline tidb-cloud/v8.1-performance-benchmarking-with-sysbench.md:1]: Preview: https://pingcap-docsite-preview-git-preview-cl-ee5c8c-oreoxmts-projects.vercel.app/tidbcloud/v8.1-performance-benchmarking-with-sysbench

File: tidb-cloud/v8.1-performance-benchmarking-with-tpcc.md
Comment (Oreoxmt) [inline tidb-cloud/v8.1-performance-benchmarking-with-tpcc.md:1]: Preview: https://pingcap-docsite-preview-git-preview-cl-ee5c8c-oreoxmts-projects.vercel.app/tidbcloud/v8.1-performance-benchmarking-with-tpcc

File: tidb-cloud/v8.1-performance-benchmarking-with-sysbench.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+- Cluster configuration:
+
+    | Node type | Node size | Node quantity | Node storage |
+    |:----------|:----------|:----------|:----------|
+    | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+    | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+### Parameter configuration
+
+The system variables [`tidb_session_plan_cache_size`](https://docs.pingcap.com/tidb/stable/system-variables#tidb_session_plan_cache_size-new-in-v710) controls the maximum number of plans that can be cached. The default value is `100`. For each workload, this document conducts tests with `tidb_session_plan_cache_size` set to `1000`:
```
Comment (hfxsd) [inline tidb-cloud/v8.1-performance-benchmarking-with-sysbench.md:32]:
```suggestion
The system variable [`tidb_session_plan_cache_size`](https://docs.pingcap.com/tidb/stable/system-variables#tidb_session_plan_cache_size-new-in-v710) controls the maximum number of plans that can be cached. The default value is `100`. For each workload, this document conducts tests with `tidb_session_plan_cache_size` set to `1000`:
```

File: tidb-cloud/v8.1-performance-benchmarking-with-sysbench.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+The TiKV parameter [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) can make log recycling effective immediately after initialization. This document conducts tests based on different workloads with the following `prefill-for-recycle` configuration:
+
+- For the `oltp_point_select` workload, use the default value of the [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) parameter:
+
+    ```yaml
+    raft-engine.prefill-for-recycle = false
+    ```
+
+- For `oltp_insert`, `oltp_read_write` , `oltp_update_index`, and `oltp_update_non_index` workloads, enable the [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) parameter:
```
Comment (hfxsd) [inline tidb-cloud/v8.1-performance-benchmarking-with-sysbench.md:50]:
```suggestion
- For `oltp_insert`, `oltp_read_write`, `oltp_update_index`, and `oltp_update_non_index` workloads, enable the [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) parameter:
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**